### ISSUE TITLE
Ethical Hosting Provider

### ISSUE THREAD

**Issue by Mitzi-Laszlo** [solid/process#215]
At the moment forum.solidproject.org and solidproject.org are hosted on AWS. 

As discussed at the Solid Team meeting with minutes on https://www.w3.org/community/solid/wiki/Meetings#2020-07-15 the intention is to move towards a third party hosting provider that is: 
* equivalent in price to AWS 
* equivalent in service convenience to AWS 
* green energy
* hosted legally and physically in Europe to be linked to GDPR without the involvement of "third countries"

Any objections to https://greenhost.net/products/hosting/? 

Any suggestions to other solutions that have the criteria mentioned above?

**Comment by csarven** [comment]
I may have previously shared https://ethical.net/resources/?resource-category=web-hosting somewhere..

I use Infomaniak (CH based) for some of my personal sites. See their "actions for sustainable, responsible growth":
https://www.infomaniak.com/en/green-hosting . I don't know how they compare to AWS on resource/pricing but worth a look.

**Comment by Mitzi-Laszlo** [comment]
Yes, I found Green Host on ethical.net. Of the web hosting options listed on ethical.net it was the only one that fit all the criteria above. 

Infomaniak looks like a good option and although not in Europe strictly it is on the adequate country listing https://ec.europa.eu/info/law/law-topic/data-protection/international-dimension-data-protection/adequacy-decisions_en

**Comment by timbl** [comment]
Any reason not to go with GreenHost?

**Comment by timbl** [comment]
What would it take to switch to GreenHost? 
[] Open GreenHost account with MIT CC
[] Clone the setup onto GreenHost
[] Switch DNS
[] Wind down old service
I assume

**Comment by almereyda** [comment]
As this issue is not yet resolved, I would like to introduce a few more suggestions to the discussion. These are derived from the [Librehosters network](https://lab.libreho.st/librehosters/directory/blob/master/directory.json), and have proven significantly stable, while grown out of grassroots, social and solidarity or cooperative environments. Some also run their own data centres, and all usually use renewable energy.

In declining order of a likely fit to your requirements, without knowing them furthen than the above:

- [Indiehosters](https://indiehosters.net/se-lancer-avec-liiibre/)  
They have a long standing record of supporting the IndieWeb, hence the name, and communicating the subject of decentralisation to 'social innovation' communities. Founder Pierre Ozoux also organised the [Labs Camp #3 at Mozilla 2015](http://2015.labs.ouishare.net/), where Michiel used to work, and Henry Story, Harry Halpin and Primavera di Phillipi were recurring guests over the years. They keep close ties with Framasoft and la Quadrature du Net and are at the core of the Commons movement in France (Assemblée Virtuelle, Remix the Commons). They are in course of migrating their containerised infrastructure into a fault-tolerant Kubernetes thing. Who doesn't remember elf Pavlik, who would also vouch for them.
- [LibreOps](https://libreops.cc/about/)  
The collective is very well rooted in Greece' activist-libertarian world, and engages against austerity and for the Commons. Their active contributors have participated in the open source sattelite SatNOGs programme, run the Athens hackerspace and also run data centres for research infrastructure with grnet.
- [Hostsharing eG](https://www.hostsharing.net/team/)
A cooperative that operates their own physical hardware with 20 years of experience in running in production. The members are actively bridging operations with civic-society initatives and participate in German discourses around 'Bits & Bäume' for sustainable use of technology. They are very well versed in finding a suitable [Service Level Agreement](https://www.hostsharing.net/service/sla-service-level-agreements/) and are known for their stability. Due to their size, they also provide multiple levels of virtualisation (shared hosting, KVM, containers) and have a larger team for support.
- [Wehost](https://weho.st/faq)
Wehost colocate in a data centre with physical access in Amsterdam and are hence at a great spot for international connectivity.

This comment is primarily motivated by offering other GDPR-hosted organisations for choice, other than for-profit companies. Librehosters and Hostsharing are both registered as legal bodies and work in a cooperative, not-for-profit manner, and the others would eventually also be able to provide individual contracts/agreements.

My proposal is to support the social and solidarity economy wherever possible, even when only directing small financial amounts in its direction to support the idealists and counter culture. *small is beautiful* (:

All are very well experienced in hosting static websites and Discourse.
